src/DynamoPackagesUI/Properties/AssemblySharedInfo.cs

```diff
@@ -0,0 +1,66 @@
+﻿using System;
```

Comment:
You don't need to copy this file to include in your project. This file should be a soft link as done in other projects.